Tests/Indicators/OptionBaseIndicatorTests.cs

```diff
@@ -102,7 +102,7 @@ protected OptionPricingModelType ParseSymbols(string[] items, bool american, out
             }
 
             expected = double.Parse(items[putColumn], NumberStyles.Any, CultureInfo.InvariantCulture);
-            acceptance = Math.Max(errorMargin * Math.Abs(expected), 1e-4);     // percentage error
+            acceptance = Math.Max(errorMargin * Math.Abs(expected), 1.5e-3);     // percentage error
```

Comment:
Let's make the percentage error a virtual property with a low default value and the greek which has a higher error rate (like `1.5e-3`) can override it, this way it's clear which greek has a better approximation or not